src/serialize.h

```diff
@@ -1103,22 +1103,36 @@ size_t GetSerializeSize(const T& t)
     return (SizeComputer() << t).size();
 }
 
-/** Wrapper that overrides the GetParams() function of a stream (and hides GetVersion/GetType). */
-template <typename SubStream, typename Params>
+/** Wrapper that overrides the GetParams() function of a stream. */
```

Comment:
re: https://github.com/bitcoin/bitcoin/pull/28929#discussion_r1497210553

> [af7f90d](https://github.com/bitcoin/bitcoin/commit/af7f90da74b935c17d986b0f4d21e4bce6f56343): Thanks for removing it, but I think it can be a separate (trivial) 4 line diff commit?

Sure, moved to separate commit